app/services/spree/retail/shopify/refunder.rb

```diff
@@ -0,0 +1,60 @@
+module Spree::Retail
+  module Shopify
+    class Refunder
+      def initialize(credited_money:, transaction:, refunder_interface: nil, can_issue_refund_policy_klass: nil, **options)
+        @credited_money = BigDecimal.new(credited_money)
+        @transaction = transaction
+        @refund_reason = options[:reason]
+        @order_id = transaction.order_id
+
+        @refunder_interface = refunder_interface || default_refunder_interface
+        @can_issue_refund_policy_klass = can_issue_refund_policy_klass || default_can_issue_refund_policy_klass
```

Comment:
Does this read any different with the `can_issue` prefix removed? Is it shorter/less repetitive or harder to discern?